test/unit-test/ota_utest.c

```diff
@@ -370,7 +370,7 @@ static void otaInitDefault()
 static void otaDeinit()
 {
     mockOSEventReset( NULL );
-    OTA_AgentShutdown( 1 );
+    OTA_Shutdown( 1 );
```

Comment:
What does `1` mean?  If it means "true" then let's pass `true` here.  And a deeper question – what does it mean to pass a value other than 1 to this API?  If different values have different meanings, should that parameter be an `enum` type or should we use `#define`s for the values?

Reply:
`OTA_Shutdown()` takes the parameter `ticksToWait`, which is the time to spend idle before the agent shuts down.
For the calls to the shutdown we have a const parameter `otaDefaultWait` [1000]
Updated this line to use the default wait.